src/Elasticsearch/Namespaces/IndicesNamespace.php

```diff
@@ -275,6 +275,25 @@ public function snapshotIndex($params = array())
     }
 
     /**
+     * @param $params array Associative array of parameters
```

Comment:
Similarly to the endpoint, we need to move this into `Client.php`, since it is part of the "main" namespace.